cmd/openshift-tests/storagecaps.go

```diff
@@ -57,17 +67,30 @@ func printStorageCapabilities(out io.Writer) {
 		return
 	}
 
-	supportsSnapshot := yamlManifest.DriverInfo.Capabilities.Snapshot && yamlManifest.SnapshotClass.FromName
-
 	fmt.Fprintln(out)
 	fmt.Fprintln(out, "Storage Capabilities (guaranteed only on full CSI test suite with 0 fails)")
 	fmt.Fprintln(out, "==========================================================================")
-	fmt.Fprintln(out, "Driver short name:                 ", yamlManifest.ShortName)
-	fmt.Fprintln(out, "Driver name:                       ", yamlManifest.DriverInfo.Name)
-	fmt.Fprintln(out, "Storage class:                     ", yamlManifest.StorageClass.FromFile)
-	fmt.Fprintln(out, "Raw block VM disks supported:      ", yamlManifest.DriverInfo.Capabilities.Block)
-	fmt.Fprintln(out, "Live migration supported:          ", yamlManifest.DriverInfo.Capabilities.RWX)
-	fmt.Fprintln(out, "VM snapshots supported:            ", supportsSnapshot)
-	fmt.Fprintln(out, "Storage-assisted cloning supported:", supportsSnapshot)
+	fmt.Fprintln(out, "Driver short name:                      ", yamlManifest.ShortName)
+	fmt.Fprintln(out, "Driver name:                            ", yamlManifest.DriverInfo.Name)
+	fmt.Fprintln(out, "Storage class:                          ", yamlManifest.StorageClass.FromFile)
+	fmt.Fprintln(out, "Supported CSI features:")
```

Comment:
Now that I think about it, it should say "Supported OpenShift / CSI features" - some of the features are OpenShift (or Kubernetes) features, not really CSI.